corehq/apps/cloudcare/views.py

```diff
@@ -461,3 +463,17 @@ def put(self, request, *args, **kwargs):
         ], bulk=False)
         access.save()
         return json_response({'success': 1})
+
+
+@login_and_domain_required
+def report_formplayer_error(request, domain):
+    data = json.loads(request.body)
+    metrics_counter('commcare.formplayer.browser_error_reports', tags={
+        'action': data.get('action'),
+        'statusText': data.get('statusText'),
+        'state': data.get('state'),
+        'status': data.get('status'),
+        'domain': domain,
+    })
+    notify_error(message='Formplayer error reported from the browser', details=data)
```

Comment:
Can this go to the formplayer project in sentry?

Reply:
Oh, hmm. I think there's not a simple way to do that from HQ. SInce it's coming through HQ and not reported directly from formplayer, I think it's fine for it to show up in the commcarehq sentry project though